app/services/encrypted_email.rb

```diff
@@ -1,24 +1,24 @@
 class EncryptedEmail
-  attr_reader :user_access_key
+  attr_accessor :user_access_key
 
-  def self.new_user_access_key
+  def self.new_user_access_key(cost = nil, key = nil)
```

Comment:
good call. that I had to keep referring to the method signature should have been my tell.